packages/navstar/build

```diff
@@ -22,7 +22,7 @@ mkdir -p $(dirname $service)
 
 navstar_service=${PKG_PATH}/dcos.target.wants/dcos-navstar.service
 mkdir -p $(dirname $navstar_service)
-cp /pkg/extra/dcos-navstar.service "$navstar_service"
+envsubst '$PKG_PATH' < "/pkg/extra/dcos-navstar.service" > "${navstar_service}"
```

Comment:
I believe `envsubst '${PKG_PATH}' ...` is more explicit per the [SHELL-FORMAT docs](https://www.gnu.org/software/gettext/manual/gettext.html#sh_002dformat) Also, what was the reason for this change? Was WorkingDirectory not being set correctly before?